--- sussen-0.3/src/Makefile.am.orig	Mon Jun 16 21:40:35 2003
+++ sussen-0.3/src/Makefile.am	Wed Jul  2 11:10:45 2003
@@ -8,11 +8,9 @@
 	$(GNOME_CFLAGS)\
 	 -I. -I/usr/include -I/usr/include/nessus -I/usr/lib/include -I/usr/include/openssl
 
-CFLAGS =\
-	 -DHAVE_CONFIG_H -DNESSUS_ON_SSL -DHAVE_SSL -DNESSUSD_CONFDIR=\"/usr/local/etc\" -DNESSUSD_STATEDIR=\"/usr/local/var/nessus\" -DNESSUSD_DATADIR=\"/usr/local/etc/nessus\" -DNESSUSD_LIBDIR=\"/usr/local/lib/nessus\" -DNESSUSD_PLUGINS=\"/usr/local/lib/nessus/plugins\" -DNESSUSD_REPORTS=\"/usr/local/lib/nessus/reports\" -DNESSUSD_SHAREDSTATEDIR=\"/usr/local/com/nessus\" -DNESSUSD_LOGDIR=\"/usr/local/var/nessus/logs\" -DPREFIX=\""$(prefix)"\" -DSYSCONFDIR=\""$(sysconfdir)"\" -DDATADIR=\""$(datadir)"\" -DLIBDIR=\""$(libdir)"\"\
-	 -Wall\
-	 -O2\
-	 -g
+CFLAGS = @CFLAGS@ \
+	 -DHAVE_CONFIG_H -DNESSUS_ON_SSL -DHAVE_SSL -DNESSUSD_CONFDIR=\"/etc/nessus\" -DNESSUSD_STATEDIR=\"/var/lib/nessus\" -DNESSUSD_DATADIR=\"/etc/nessus\" -DNESSUSD_LIBDIR=\"/usr/lib/nessus\" -DNESSUSD_PLUGINS=\"/usr/lib/nessus/plugins\" -DNESSUSD_REPORTS=\"/usr/lib/nessus/reports\" -DNESSUSD_SHAREDSTATEDIR=\"/var/lib/nessus\" -DNESSUSD_LOGDIR=\"/var/lib/nessus/logs\" -DPREFIX=\""$(prefix)"\" -DSYSCONFDIR=\""$(sysconfdir)"\" -DDATADIR=\""$(datadir)"\" -DLIBDIR=\""$(libdir)"\"\
+	 -Wall
 
 bin_PROGRAMS = sussen
 
